src/browser.ts

```diff
@@ -1,10 +1,9 @@
 export { default as ApolloProvider } from './ApolloProvider';
 export { default as graphql, withApollo, InjectedGraphQLProps } from './graphql';
 
-
 // expose easy way to join queries from redux
 export { compose } from 'redux';
 
 // re-exports of close dependencies.
-export { default as ApolloClient, createNetworkInterface, createBatchingNetworkInterface } from 'apollo-client';
+export * from 'apollo-client';
```

Comment:
`ApolloClient` is also a named export from `apollo-client` (as well as the default export) so it should make no difference.

Ref: https://github.com/apollographql/apollo-client/blob/master/src/index.ts#L121